### ISSUE TITLE
Graphviz sneakily installs files in TCL subdirectories 

### ISSUE THREAD

**Issue by damianam** [hpcugent/easybuild-easyconfigs#3547]
In particular in `$EBROOTTCL/lib64/tcl8.6/graphviz/`. That causes Python to fail installing with:

`Multiple library subdirectories found for Tcl in $EBROOTTCL: lib, lib64`

**Comment by boegel** [comment]
urgh, thanks for reporting this @damianam!

**Comment by boegel** [comment]
I looked into this, and it's even worse: it's also installing Graphviz bindings in the Python and Perl installation directories:

* `$EBROOTPYTHON/lib/python2.7/site-packages/_gv.so`
* `$EBROOTPERL/lib/perl5/site_perl/5.20.0/x86_64-linux-thread-multi/gv.so`

This is done via the `install-data-hook` target in `tclpkg/Makefile`:

```
# Installation of softlinks into non-owned directories is done as a                                                                                                                                           
# separate step, with soft failures, so that a developer can do:                                                                                                                                              
#    ./configure --prefix=$HOME/graphviz; make; make install                                                                                                                                                  
# without root priviledges.                                                                                                                                                                                   
install-data-hook: 
```

It looks hard to avoid, without patching that Makefile...

**Comment by damianam** [comment]
I suspected something like that too, but haven't checked. I suppose it is the same for R, OCaml, etc. Nasty.

**Comment by boegel** [comment]
fixed in #3556, let's follow up there...
